.changeset/eleven-poems-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@smithy/util-waiter": minor
+---
+
+modify the waitUntilObjectExists Validation for minDelay
```

Comment:
The changeset message should say this validation is fixed for all waiters.

Also, don't capitalize the word Validation.